lib/id3c/cli/redcap.py

```diff
@@ -254,15 +254,30 @@ class Record(dict):
     Must be constructed with a REDCap :py:class:``Project``, which is stored as
     the ``project`` attribute and used to set the ``id`` attribute storing the
     record's primary id.
+
+    Note that event_name and repeat_instance might not be populated because their
+    corresponding fields (redcap_event_name and redcap_repeat_instance)
+    won't be returned by the API if the request includes the 'fields' parameter
+    but not the record_id field.
```

Comment:
```suggestion
    Note that the ``event_name`` and ``repeat_instance`` attributes might not
    be populated because their corresponding fields (``redcap_event_name`` and
    ``redcap_repeat_instance``) won't be returned by the API if the request
    includes the ``fields`` parameter but not the primary record id field.
```

Tweaks to make this docstring addition 1) conform to the rest of the codebase's use of rST syntax and 2) be more precise about `record_id`.